lib/redcloth.rb

```diff
@@ -36,10 +36,3 @@ def self.include(*args)
   end
   
 end
-
-begin
-  require 'erb'
-  require 'redcloth/erb_extension'
-  include ERB::Util
```

Comment:
This like also includes `ERB::Util` in the Object class. Even that `t` method is not removed I think this line should be removed.